src/OnboardingSPA/components/LivePreview/BlockPreview/stylesheet.scss

```diff
@@ -17,3 +21,51 @@
 		}
 	}
 }
+
+.is-skeleton {
```

Comment:
If this is in the live-preview component this needs to go inside `.live-preview-styling`, I understand we use this CSS globally but this code globally here can create confusion.